tools/rustfmt/srcs/main.rs

```diff
@@ -14,52 +18,46 @@ fn main() {
     apply_rustfmt(&options, &targets);
 }
 
-/// Perform a `bazel` query to determine a list of Bazel targets which are to be formatted.
-fn query_rustfmt_targets(options: &Config) -> Vec<String> {
-    // Determine what packages to query
-    let scope = match options.packages.is_empty() {
-        true => "//...:all".to_owned(),
-        false => {
-            // Check to see if all the provided packages are actually targets
-            let is_all_targets = options
-                .packages
-                .iter()
-                .all(|pkg| match label::analyze(pkg) {
-                    Ok(tgt) => tgt.name != "all",
-                    Err(_) => false,
-                });
-
-            // Early return if a list of targets and not packages were provided
-            if is_all_targets {
-                return options.packages.clone();
+/// The edition to use in cases where the default edition is unspecified by Bazel
+const FALLBACK_EDITION: &str = "2018";
+
+/// Determine the Rust edition to use in cases where a target has not explicitly
+/// specified the edition via an `edition` attribute.
+fn get_default_edition() -> &'static str {
+    // TODO: THe `RUST_DEFAULT_EDITION` should be set by the `rust_toolchain`
```

Comment:
When isn't this? What would break if we just did `env!("RUST_DEFAULT_EDITION")` rather than the option/fallback stuff? It seems like there's a _lot_ of code here to handle this case?